src/lib/http/http_server.pl

```diff
@@ -121,37 +121,44 @@
 send_response(ResponseHandle, http_response(StatusCode0, text(ResponseText), ResponseHeaders0)) :-
     default(StatusCode0, 200, StatusCode),
     maplist(map_header_kv_2, ResponseHeaders, ResponseHeaders0),
-    '$http_answer'(ResponseHandle, StatusCode, ResponseHeaders, ResponseStream),
-    call_cleanup(
-	format(ResponseStream, "~s", [ResponseText]),
-	close(ResponseStream)
+    '$http_answer'(ResponseHandle, StatusCode, ResponseHeaders, ResponseStream0),
+    open(stream(ResponseStream0), write, ResponseStream, [type(text)]),
+    catch(
+	call_cleanup(format(ResponseStream, "~s", [ResponseText]),close(ResponseStream)),
+	error(existence_error(stream, _), _),
+	true
```

Comment:
Is there a reason to suppress this exception if it occurs?

It seems it could be interesting to know whether sending failed!

Reply:
This might happen if the HTTP request was canceled by the client. At that point, we can safely ignore the error, there's no point in sending it again since we lost the connection, and don't stop the server.